urdf2webots/parserURDF.py

```diff
@@ -877,4 +882,9 @@ def parseGazeboElement(element, parentLink, linkList):
                         lidar.noise = float(noiseElement.getElementsByTagName('stddev')[0].firstChild.nodeValue)
                         if lidar.maxRange:
                             lidar.noise /= lidar.maxRange
+                # minRange and near default values are 0.01 in Webots; ensure constraint near <= minRange
+                if not lidar.near and lidar.minRange < 0.01:
+                    lidar.near = lidar.minRange
+                if not lidar.minRange and lidar.near > 0.01:
+                    lidar.minRange = lidar.near
```

Comment:
That is better, we just need to also ensure the constraint when both values are set.

I think it is better to increase `minRange` rather than decrease `near`, as a too low `near` might render parts of the lidar during measurements.

```suggestion
                # minRange and near default values are 0.01 in Webots; ensure constraint near <= minRange
                if lidar.near and lidar.minRange and lidar.near > lidar.minRange:
                    lidar.minRange = lidar.near
                    print('The "minRange" value cannot be strictly inferior to the "near" value for a lidar, "minRange" has been set to the value of "near".')
                elif not lidar.near and lidar.minRange < 0.01:
                    lidar.near = lidar.minRange
                elif not lidar.minRange and lidar.near > 0.01:
                    lidar.minRange = lidar.near
```